tools/dia_dll.py

```diff
@@ -44,7 +44,7 @@ def GetDiaDll():
   msvs_version = vs_toolchain.GetVisualStudioVersion()
 
   if bool(int(os.environ.get('DEPOT_TOOLS_WIN_TOOLCHAIN', '1'))):
-    dia_path = os.path.join(win_sdk_dir, '..', 'DIA SDK', 'bin', 'amd64')
+    dia_path = os.path.join(win_sdk_dir, '..', '..', 'DIA SDK', 'bin', 'amd64')
```

Comment:
The depot tools moved the Windows 10 SDK one level deeper. Previous depot tools layout:

```
\- DIA SDK
\- Windows Kits
 \- ... # This is the location of `DEPOT_TOOLS_WIN_TOOLCHAIN`
```

New depot tools layout:

```
\- DIA SDK
\- Windows Kits
  \- 10 # New folder
    \- ... # This is the location of `DEPOT_TOOLS_WIN_TOOLCHAIN`
```

Reply:
Oh whoops, I merged before seeing this comment. I'll add a comment in a follow-up PR for the ANGLE roll